packages/lib-utils/src/components/getting-started-card/GettingStartedCard.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Button,
+  Card,
+  CardActions,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import CloseIcon from '@patternfly/react-icons/dist/js/icons/close-icon';
+import * as React from 'react';
+import '@patternfly/react-styles/css/utilities/Display/display.css';
+import '@patternfly/react-styles/css/utilities/Spacing/spacing.css';
+import { useLocalStorage } from '../../hooks';
+
+type GettingStartedCardProps = {
+  cardClassName?: string;
+  imgAlt?: string;
+  imgClassName?: string;
+  imgSrc?: string;
+  isDismissable?: boolean;
+  localStorageKey: string;
+  title: string;
+};
+
+const LOCAL_STORAGE_KEY = 'getting-started-card';
+
+const GettingStartedCard: React.FC<GettingStartedCardProps> = ({
+  cardClassName,
+  children,
+  imgAlt = '',
+  imgClassName,
+  imgSrc,
+  isDismissable = true,
+  localStorageKey,
+  title,
+}) => {
+  const [storageKeys, setStorageKeys] =
+    useLocalStorage<{ [key: string]: boolean }>(LOCAL_STORAGE_KEY);
+
+  const keys = storageKeys && typeof storageKeys === 'object' ? storageKeys : {};
+  const isDismissed = keys[localStorageKey];
+
+  return !isDismissed ? (
+    <Card className={cardClassName}>
+      <Split>
+        {imgSrc && (
+          <SplitItem className={imgClassName}>
+            <img src={imgSrc} alt={imgAlt} />
```

Comment:
We ended up adding a default size for the img in hac-dev because it was the most common size

https://github.com/openshift/hac-dev/blob/342c26553fdcc9840fc1099c763f1349892f7627/src/components/GettingStartedCard/GettingStartedCard.scss#L2-L5

But maybe it's not needed here?

Reply:
If the SVG has `width` and `height` attributes within, a size is not needed as it is embedded in the SVG.  I noticed the SVG used in hac-dev is missing these attributes.

Reply:
Did not know that, thanks.
Those svgs are just temporary placeholders so I think they're fine for now.